src/service-providers/vault-level/vault-level.provider.ts

```diff
@@ -34,6 +35,10 @@ class VaultProvider extends ServiceProviderBase {
   public reloadDs(params: IAdmReloadDsArguments) {
     return this.post<string>('adm_reload_ds', params).then(res => res.data)
   }
+
+  public dropDs(params: IAdmDropDsArguments) {
+    return this.post<any>('adm_drop_ds', params).then(res => res.data)
```

Comment:
Maybe it returns some string? At least "OK" or something

Reply:
Added type "string" instead of "any"